playbooks/roles/deploy-osh/templates/horizon.yaml.j2

```diff
@@ -1,2 +1,27 @@
 ---
+{% if developer_mode %}
```

Comment:
We should probably add an override here to point to your built image if developer mode.

Reply:
out of scope for this patch? Separate patch better?